tests/helpers.py

```diff
@@ -374,6 +374,30 @@ def create_storage_class(
     return create_resource(**sc_data)
 
 
+def create_mcg_storageclass(reclaim_policy=constants.RECLAIM_POLICY_DELETE, sc_name=None):
```

Comment:
I think you can utilize the already existing create_storageclass() function we have in helpers.py and tp pass it the MCG SC yaml

Reply:
create_storageclass() in helpers has certain default parameters like secret_name, interface_name which is not required to create a mcg storage class.